libusb/os/windows_winusb.c

```diff
@@ -2754,8 +2770,36 @@ static int winusbx_release_interface(int sub_api, struct libusb_device_handle *d
 	if (!HANDLE_VALID(winusb_handle))
 		return LIBUSB_ERROR_NOT_FOUND;
 
+	int initialized_iface;
+	if (priv->usb_interface[iface].is_associated_interface)	{
```

Comment:
This will have to be `number_associated_interfaces != 0` once my version of 965 has landed in master.

Reply:
👍 I will change it once it's merged

Reply:
Good. Above change can be done already in any case, it is equivalent.